modules/velero-obtaining-by-accessing-binary.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/troubleshooting.adoc
+
+:_content-type: PROCEDURE
+[id="velero-obtaining-by-accessing-binary_{context}"]
+= Accessing the Velero binary in the Velero deployment in the cluster
+
+You can use a shell command to access the Velero binary in the Velero deployment in the cluster.
+
+.Prerequisites
+
+* our Data Protection Application custom resource has a status of `Reconcile complete`.
+.Procedure
```

Comment:
what's the procedure part? seems like it has a wrong context

Reply:
Fixed; thanks.